### PR TITLE
Add `sql_query_insert.PqConnection()`

### PR DESCRIPTION
**Pull request by mgirlich**
This method uses `ON CONFLICT DO NOTHING`

### CODE REVIEW

**Comment by hadley** [comment]
Are these PRs ready for review?

**Comment by mgirlich** [comment]
@hadley Yes, they are ready to review 😄

**Comment by hadley** [comment]
@mgirlich if you want me to respond faster, can you please request a review from me? That way I get a notification about it.

File: R/backend-postgres.R [hunk 1]
```diff
@@ -250,6 +250,34 @@ sql_query_explain.PqConnection <- function(con, sql, format = "text", ...) {
...
+  parts <- rows_prep(con, x_name, y, by, lvl = 0)
+
+  insert_cols <- escape(ident(colnames(y)), collapse = ", ", parens = TRUE, con = con)
+  by_sql <- escape(ident(by), parens = TRUE, collapse = ", ", con = con)
+  clauses <- list(
+    sql_clause_insert(con, insert_cols, x_name),
+    sql_clause_select(con, sql("*")),
+    sql_clause_from(parts$from),
+    # `WHERE true` is required for SQLite
+    sql("WHERE true"),
```
Comment (hadley) [inline R/backend-postgres.R:271]: Can remove here?

File: R/backend-postgres.R [hunk 2]
```diff
@@ -250,6 +250,34 @@ sql_query_explain.PqConnection <- function(con, sql, format = "text", ...) {
 #' @export
 sql_query_explain.PostgreSQL <- sql_query_explain.PqConnection
 
+#' @export
+sql_query_insert.PqConnection <- function(con, x_name, y, by,
```
Comment (hadley) [inline R/backend-postgres.R:254]: Do you mind taking a quick look over the other `sql_query_insert` methods and see if there are any opportunities to reuse code?

**Review by hadley — APPROVED** [review]

**Pull request merged**